### PR TITLE
DB-2228: change private to Forget.

### PR DESCRIPTION
**Pull request by alver-cliqz**

### CODE REVIEW

File: l10n/de/browser/browser/aboutPrivateBrowsing.ftl [hunk 1]
```diff
@@ -2,27 +2,27 @@
...
 about-private-browsing-search-placeholder = Das Web durchsuchen
 about-private-browsing-info-bookmarks = Lesezeichen
-about-private-browsing-info-title = Dies ist ein privates Fenster
+about-private-browsing-info-title = Dies ist ein Vergessen Fenster
 about-private-browsing-info-searches = Suchanfragen
 about-private-browsing-info-downloads = Downloads
-private-browsing-title = Privates Surfen
+private-browsing-title = Vergessen Modus
 about-private-browsing-info-saved = { -brand-short-name } wird <strong>Folgendes speichern</strong>:
 about-private-browsing-info-myths = Häufige Missverständnisse über das Surfen im Privaten Modus
```
Comment (tsl143) [inline l10n/de/browser/browser/aboutPrivateBrowsing.ftl]: Privaten Modus - Vergessen Modus

File: mozilla-release/browser/locales/en-US/browser/aboutPrivateBrowsing.ftl [hunk 2]
```diff
@@ -2,27 +2,27 @@
...
 about-private-browsing-search-placeholder = Search the Web
 about-private-browsing-info-bookmarks = bookmarks
-about-private-browsing-info-title = You’re in a Private Window
+about-private-browsing-info-title = You’re in a Forget Window
 about-private-browsing-info-searches = searches
 about-private-browsing-info-downloads = downloads
-private-browsing-title = Private Browsing
+private-browsing-title = Forget Mode
 about-private-browsing-info-saved = { -brand-short-name } <strong>will save</strong> your:
 about-private-browsing-info-myths = Common myths about private browsing
```
Comment (tsl143) [inline mozilla-release/browser/locales/en-US/browser/aboutPrivateBrowsing.ftl]: Can we say `Common myths about forget mode`?

**Comment by alver-cliqz** [comment]
All this texts used on Firefox about:privatebrowsing page, we have our own version, which have nothing in common with Firefox's version. So, I decide not to change it

**Comment by alver-cliqz** [comment]
Updated to reflect changes everywhere (for consistent)

**Pull request merged**